styles/_helpers.scss

```diff
@@ -4,5 +4,5 @@
 }
 
 .is-hidden {
-  display: none;
+  display: none !important;
```

Comment:
Well, tbh I didn't add the `!important` because of that. I had no choice :/ Like @nerik I am not fan of using it unless you have no choice. Otherwise, thank you both for the feedback!